### PR TITLE
Feature/users profile

### PR DESCRIPTION
**Pull request by Gladdy17**

### CODE REVIEW

File: pawsitive/db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,24 @@
...
+    t.string "address"
+    t.string "country"
+    t.string "city"
+    t.string "state"
+    t.string "zip_or_postcode"
+    t.date "birthday"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_accounts_on_user_id"
+  end
```
Comment (lincoln1883) [inline pawsitive/db/schema.rb:29]: Hi @Gladdy17, now that we add the extra columns to the users table this accounts table should be remove.

File: pawsitive/app/views/users/profile.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <% if current_user.role == 'member' %>
-  <%= render 'users/member_profile' %>
+    <%= render 'users/member_profile' %>
+  <% else %>
+    <%= render 'users/users_profile' %>
```
Comment (lincoln1883) [inline pawsitive/app/views/users/profile.html.erb:4]: the actual profile page/template is missing.

Screenshot from 2024-05-22 23-27-49

File: pawsitive/db/schema.rb (hunk 1 above)
Comment (Gladdy17) [reply pawsitive/db/schema.rb:29]: this one i will remove in the current branch i am working on

File: pawsitive/app/views/users/profile.html.erb (hunk 2 above)
Comment (Gladdy17) [reply pawsitive/app/views/users/profile.html.erb:4]: I just pushed changes can you try again

File: pawsitive/app/models/user.rb [hunk 3]
```diff
@@ -3,9 +3,10 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-         
+
   has_many_attached :photos, service: :amazon, dependent: :destroy
   has_one_attached :avatar, service: :amazon, dependent: :destroy
+  has_one :account
```
Comment (lincoln1883) [inline pawsitive/app/models/user.rb]: remove this validation.

**Review by lincoln1883 — APPROVED** [review]
Hey @Gladdy17, good job on the client profile page, it time to merge.

**Pull request merged**